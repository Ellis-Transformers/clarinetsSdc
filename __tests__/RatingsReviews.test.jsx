import renderer from 'react-test-renderer';
import ReviewsList from '../src/dist/Components/Ratings_Reviews/ReviewsList.jsx';

it('renders component when given an array', () => {
  const component = renderer.create(
    <ReviewsList shownReviews={[]}/>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});