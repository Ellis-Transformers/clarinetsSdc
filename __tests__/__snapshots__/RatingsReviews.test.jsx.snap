// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`displays a list of review entries 1`] = `
<div
  style={
    {
      "maxHeight": "50%",
      "overflow": "auto",
    }
  }
/>
`;

exports[`renders component when given an array 1`] = `
[
  <style
    dangerouslySetInnerHTML={
      {
        "__html": ".css-17bk85d{max-height:75vh;overflow:auto;}",
      }
    }
    data-emotion="css 17bk85d"
  />,
  <div
    className="MuiBox-root css-17bk85d"
  />,
]
`;
